README.md

```diff
@@ -90,3 +90,9 @@ If you don't have one, you can make the following changes to make GreeyBear spin
 
 1) In ```docker/env_file```, set the variable ```ELASTIC_ENDPOINT``` to ```http://elasticsearch:9200```.
 2) Add ```:docker/elasticsearch.yml``` to the last defined ```COMPOSE_FILE``` variable or uncomment the ```# local development with elasticsearch container``` block in ```.env``` file.
```

Comment:
I think we could reduce the size of the README here because the details were moved to the documentation.

we could remove the "Feeds part" and the "How to execute it locally for testing"

Reply:
I have removed the "available feeds" part but kept the "public feeds" same over there as they  give base idea on how the project demo look like